test/unit/models/transaction.test.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * This eslint rule is disabled because of https://github.com/nodesecurity/eslint-plugin-security/issues/21
+ * It gives linting errors in testing the DB data with keys from fixtures
+ */
+/* eslint-disable security/detect-object-injection */
+
+const chai = require('chai')
+const { expect } = chai
+
+const cleanDb = require('../../utils/cleanDb')
+const transactionModelsImpl = require('../../../models/transaction')
+const transactionDataArray = require('../../fixtures/transaction/transactions')()
+const firestore = require('../../../utils/firestore')
+const transactionsModel = firestore.collection('transaction')
+
+/**
+ * Test the model functions and validate the data stored
+ */
+
+describe('Transactions', function () {
+  before(function () {
+    transactionDataArray.forEach(async (element, index) => {
+      await transactionsModel.add(transactionDataArray[index])
+    })
+  })
+  after(async function () {
+    await cleanDb()
+  })
+
+  describe('fetchTransactionsByUserId using transaction route and get() call', function () {
```

Comment:
These tests are for `GET /transactions/:username` right?
Please check the users tests to know how we're displaying the messages in terminal. Yes, this text will appear in terminal when we run tests and that's why I'm after keeping it nice and consistent

Reply:
Oops, these are unit tests. Please check how we're writing messages for unit tests for users then

Reply:
> Oops, these are unit tests. Please check how we're writing messages for unit tests for users then

Yes sure. We are writing function names in unit tests.

Reply:
Done